lib/pf/Switch/Meraki/MS220_8.pm

```diff
@@ -0,0 +1,145 @@
+package pf::Switch::Meraki::MS220_8;
+
+=head1 NAME
+
+pf::Switch::Meraki::MS220_8
+
+=head1 SYNOPSIS
+
+The pf::Switch::Meraki::MS220_8 module implements an object oriented interface to
+manage the connection with MS220_8 switch model.
+
+=head1 STATUS
+
+Developed and tested on a MS220_8P (P standing for PoE) switch
+
+=head1 BUGS AND LIMITATIONS
+
+The firmware allow only for VLAN enforcement at the moment. We cannot push the predfine policies from PacketFence.
+
+=head2 Cannot reevaluate the access
+
+There is currently no way to reevaluate the access of the device.
+There is neither an API access or a RADIUS disconnect that can be sent either to the AP or to the controller.
+
+=cut
+
+use strict;
+use warnings;
+
+use base ('pf::Switch');
+
+use pf::constants;
+use pf::config;
+use pf::util;
+use pf::node;
+
+
+=head1 SUBROUTINES
+
+=cut
+
+# CAPABILITIES
+# access technology supported
+sub description { 'Meraki switch MS220_8' }
+sub supportsWiredMacAuth { return $TRUE; }
+sub supportsWiredDot1x { return $TRUE; }
+sub supportsExternalPortal { return $FALSE; }
+sub supportsWebFormRegistration { return $FALSE }
+
+=head2 getVersion - obtain image version information from switch
+
+=cut
+
+sub getVersion {
+    my ($self) = @_;
+    my $logger = $self->logger;
+    $logger->info("we don't know how to determine the version through SNMP !");
+    return '1';
+}
+
+=head2 parseUrl
+
+This is called when we receive a http request from the device and return specific attributes:
+
+client mac address
+SSID
+client ip address
+redirect url
+grant url
+status code
+
+=cut
+
+sub parseUrl {
```

Comment:
If it only supports VLAN assignment as you state above, why do you put this here